trigger/host_spawn.go

```diff
@@ -170,7 +170,10 @@ func (t *spawnHostStateChangeTriggers) spawnHostStateChangeOutcome(sub *event.Su
 	if !t.host.UserHost {
 		return nil, nil
 	}
-
+	if t.event.EventType == event.EventHostStarted && t.data.Successful && !t.host.Provisioned {
```

Comment:
if the event processing job runs after the host has already been provisioned might the user still get two notifications?

Reply:
Maybe, but we should handle for the case that notifications are being processed correctly--in general, this will prevent multiple notifications (I've looked through my slack and I've only seen notifications in the correct order if that helps).

Reply:
I don't think it would be incorrect for it to be processed after the host got provisioned (it would just depend how long it takes Amboy to get to processing the event vs how long it takes the host to get provisioned)